rocks-db/src/dump_client.rs

```diff
@@ -435,20 +460,27 @@ impl Storage {
             if data_channel.is_empty() {
                 continue;
             } else if let Ok(k) = data_channel.try_recv() {
+                let start = Instant::now();
+
                 if let Err(e) = writer.serialize(k).map_err(|e| e.to_string()) {
                     error!(
                         "Error while writing data into {:?}. Err: {:?}",
                         file_and_path.1, e
                     );
                 }
+
+                synchronizer_metrics.set_file_write_time(start.elapsed().as_millis() as f64);
```

Comment:
I'd suggest using a counter of writes per file, or making it a histogram per file